src/ui/components/ObjectLabel.vue

```diff
@@ -42,6 +42,10 @@ export default {
         navigateToPath: {
             type: String,
             default: undefined
+        },
```

Comment:
If we capture the event at the tree-item level, we wont have to stop propagation, and we wont need to pass down this prop.

Object level will pretty much stay the same. Tree item will have a context-click.capture event listener, which will call the listener at the capture stage of an event, you can then event.stopPropagation() to stop the event from reaching the object label.

Reply:
Good call! Updated.